lib/fuzzy_tools/index.rb

```diff
@@ -16,6 +16,11 @@ def find(query)
       result
     end
 
+    def find_with_score(query)
+      result, score = unsorted_scored_results(query).max_by { |doc, score| [score, document_attribute(doc)] }
+      [result, score]
+    end
```

Comment:
why don't you reimplement `find` in terms of `find_with_score`?
i.e. something like:
```
def find(query)
  find_with_score(query).first
end
```

Reply:
Thank you :) all done